apps/studio/src/features/sign-in/components/UnableToUseSignpassModal.tsx

```diff
@@ -0,0 +1,56 @@
+import NextLink from "next/link"
+import {
+  Link,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react"
+
+interface UnableToUseSignpassModalProps {
+  isOpen: boolean
+  onClose: () => void
+}
+
+export const UnableToUseSignpassModal = ({
+  isOpen,
+  onClose,
+}: UnableToUseSignpassModalProps): JSX.Element => {
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+
+      <ModalContent>
+        <ModalHeader mr="3.5rem">
+          Can’t use Singpass to authenticate?
+        </ModalHeader>
+
+        <ModalCloseButton size="lg" />
+
+        <ModalBody>
+          <Text>
+            You can only access Isomer Studio with Singpass. If you need to
+            access Isomer Studio urgently, ask your colleagues to help you.
+            <br />
+            <br />
```

Comment:
question - does this show as text or 2 line breaks?

Reply:
Two line breaks, I'm following what design did for Figma lol